### PR TITLE
Issue #1 Linking a pull request to an issue

### LINKED ISSUE #1: Hi
@Jian-Min-Huang 

Nice to meet you.

### PR DESCRIPTION
**Pull request by Jing-Teng**
Linking a pull request to #1

### CODE REVIEW

File: [中文] GitHub 學習資源.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[中文] GitHub 學習資源
```
Comment (Jian-Min-Huang) [inline [中文] GitHub 學習資源.md:1]: wow, awesome

**Comment by Jian-Min-Huang** [comment]
if have review progress then you will commit and push many times

and finally i will click the merge pull request button

**Comment by Jian-Min-Huang** [comment]
@Jing-Teng click!

**Comment by Jing-Teng** [comment]
I learned something. Thank you.

**Pull request merged**